src/store/modules/RiskScoringModule.js

```diff
@@ -0,0 +1,15 @@
+import { VuexModule, Module, Action } from 'vuex-class-modules';
+import riskScoringService from '@/service/riskScoring';
+
+@Module({ generateMutationSetters: true })
+class RiskScoringModule extends VuexModule {
+  /**
+   * @return {Promise<void>}
+   */
+  @Action
+  async sendUserMetrics() {
+    await riskScoringService.sendUserMetrics();
```

Comment:
Do we really need module instead controller? Module needs when we have state, but this module has not

Reply:
Commonly we use only private controllers, if something global - then modules. This is global. One module instead 3-4 private controllers with same code.

Reply:
We can also use global controller. @kazurus am I right?

Reply:
If it is acceptable, then yes 🙂 but generally we have only WalletController in auth, all remains - private or store. So this is uncommon approach.

Reply:
We should use global controller instead store. We have similar in Vault.

Reply:
what about don't use store or global controller.
and use just single `scoringService` and call them not from components, but from controllers (backController, frontOnlyController) in places, where we need `sendUserMetrics`?
@endpass/frontend ?

Reply:
Good idea

Reply:
Agreed